exasol_transformers_extension/udfs/models/sequence_classification_single_text_udf.py

```diff
@@ -54,7 +55,10 @@ def execute_prediction(self, model_df: pd.DataFrame) -> list[list[dict[str, Any]
         :return: List of dataframe includes prediction details
         """
         sequences = list(model_df["text_data"])
-        results = self.last_created_pipeline(sequences, return_all_scores=True)
+        # todo apparently top_k is now supported. so we could also solve the ranks ALL/Highest like for example in filling mask udf
```

Comment:
Not sure, if I get exactly what you are asking. We don't have exactly topk , topk is in so far, similar , that you can implement highest with topk. However, to implement ALL with it, you would need to know the number of labels.

Reply:
top_k=None gives you results for for all labels. but you need to know how many you get and sort the results accordingly in both ways. its only the question if we want to let the pipeline sort out the unwanted results, or do it ourselves. which influences how we approach the combining of input and result